### PR TITLE
Added ManualExpenseModal

### PR DESCRIPTION
**Pull request by GhazalJamil**
### What is the purpose of this PR?

Changed CashExpenseModal to ManualExpenseModal so now it's possible to add bank transactions manually

### How did you verify your code is stable and works correctly?

- Manual test

### CODE REVIEW

File: components/ManualExpenseModal.tsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  Tooltip,
+  Switch,
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  SelectChangeEvent,
+  Grid,
+  InputAdornment,
+} from "@mui/material";
```
Comment (kamka427) [inline components/ManualExpenseModal.tsx:19]: We might have to ditch mui later for better responsivity. Good solution still.

File: components/ManualExpenseModal.tsx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+import AttachMoneyIcon from "@mui/icons-material/AttachMoney";
+
+interface ExpenseForm {
```
Comment (kamka427) [inline components/ManualExpenseModal.tsx:22]: Please move interfaces to types.ts with the naming convention.

**Review by kamka427 — COMMENTED** [review]
Great work

File: .vscode/settings.json
Comment (kamka427) [inline .vscode/settings.json:1]: Please remove this file or add it to gitignore.

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (MilosAlex) [inline .vscode/settings.json:1]: Add this file to gitignore

File: components/Header.tsx [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
+//import AddCashExpenseModal from "./AddCashExpenseModal";
+import ManualExpenseModal from "./ManualExpenseModal";
 
 export default function Header() {
   return (
     <header className="w-full flex flex-row gap-16 bg-slate-300">
       <h1 className="text-4xl font-bold ml-[0.5rem] text-slate-500 ">Header</h1>
       <div className="ml-auto"></div>
-      <AddCashExpenseModal />
+      {/* <AddCashExpenseModal /> */}
```
Comment (MilosAlex) [inline components/Header.tsx:11]: Please remove the not needed comments

**Review by MilosAlex — COMMENTED** [review]
Great PR, here are my ideas for improvement:

**Review by kamka427 — APPROVED** [review]
Thank you for the fixes!

**Review by MilosAlex — APPROVED** [review]
Thanks for resolving the comments!

**Pull request merged**